### PR TITLE
Extend process.env when spawning dotnet child process. Fixes #9 

### LINKED ISSUE #9: npm start fails
At least on my OSX install the following command fails.  I have attached all the information I know

the mentioned file " /Users/mwatts/tmp/try.aspnet-starter/MyApp/npm-debug.log" does not exist.

```
>npm start

[21:18:40] Starting 'start'...
Time: 5156ms
                           Asset     Size  Chunks             Chunk Names
    main.js?cbd653dd2e9a56f5e8e9   938 kB       0  [emitted]  main
main.js.map?cbd653dd2e9a56f5e8e9  1.08 MB       0  [emitted]  main
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn dotnet ENOENT
    at exports._errnoException (util.js:1007:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)

npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/Cellar/node/6.2.1/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.2.1
npm ERR! npm  v3.9.6
npm ERR! code ELIFECYCLE
npm ERR! app@1.0.0 start: `node tools/start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the app@1.0.0 start script 'node tools/start'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the app package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node tools/start
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs app
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls app
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mwatts/tmp/try.aspnet-starter/MyApp/npm-debug.log
```

### PR DESCRIPTION
**Pull request by kmees**
I has the same Issue as in #9. This should fix the `Error: spawn dotnet ENOENT`

### CODE REVIEW

**Comment by koistya** [comment]
Looks good, thanks! I would just replace `object-extend` with a more popular `extend` module. Is there any reason why do you prefer `object-extend`?

**Comment by koistya** [comment]
@kmees I just checked, there is an existing dependency `extend` that can be used for that I believe.

File: run.js [hunk 1]
```diff
@@ -162,9 +163,9 @@ tasks.set('start', () => {
       const options = {
         cwd: path.resolve(__dirname, './server/'),
         stdio: ['ignore', 'pipe', 'inherit'],
-        env: {
+        env: extend(process.env, {
           ASPNETCORE_ENVIRONMENT: 'Development',
-        },
+        }),
```
Comment (frenzzy) [inline run.js]: why not just

Comment (frenzzy) [inline run.js]: why not just
```js
env: { ...process.env, ASPNETCORE_ENVIRONMENT: 'Development' }
// or
env: Object.assign({}, process.env, { ASPNETCORE_ENVIRONMENT: 'Development' })
```

**Pull request merged**